tailwind.config.cjs

```diff
@@ -77,6 +77,8 @@ module.exports = {
         // If we don't know the status, use gray
         'unknown': tailwindColors.gray[100],
 
+        'connected': tailwindColors.green[600],
+        'disconnected': tailwindColors.gray[500],
```

Comment:
we should move all the colors to the color-registry so we can redefine the colors by a theme extension

Reply:
reference: https://github.com/containers/podman-desktop/pull/7526#discussion_r1632770637

will need it for this PR in particular: https://github.com/containers/podman-desktop/pull/7526

Reply:
@benoitf would you be ok merging this, and I take a separate issue to move status colors to registry?

Reply:
yes

Reply:
Thanks, opened #7581 and self assigned. 👍🏼